### PR TITLE
Enable dls progress controller #4072

### PR DESCRIPTION
**Pull request by clehene**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by clehene** [comment]
Note that the capability in azure-data-lake-store-python will only be available with the next release. I'm not familiar with the process around that

File: src/command_modules/azure-cli-dls/HISTORY.rst [hunk 1]
```diff
@@ -2,9 +2,14 @@
 
 Release History
 ===============
+0.0.12
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-dls/HISTORY.rst]: This should be 'unreleased'
https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log

Comment (clehene) [inline src/command_modules/azure-cli-dls/HISTORY.rst]: @derekbekoe done

**Comment by clehene** [comment]
@derekbekoe I made the changes, please take another look

**Comment by derekbekoe** [comment]
@ro-joowan can you take a look?

**Comment by clehene** [comment]
@ro-joowan here's the relevant commit in data-lake-store-python https://github.com/Azure/azure-data-lake-store-python/commit/11f23b9ce381ad6353ad7019e3313a2e97c34aa2
ping @begoldsm

**Comment by clehene** [comment]
@derekbekoe  any other options for reviewers? :)

**Comment by ro-joowan** [comment]
Hi @clehene, @brettcannon will take a look at this.  Sorry for the delay!

**Pull request merged**